droidlet/lowlevel/minecraft/pyworld/item.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+from droidlet.base_util import Pos
+from droidlet.lowlevel.minecraft.pyworld.utils import make_pose
+
+
+class GettableItem:
+    def __init__(self, typeName, entityId=None, pos=None, idm=(0, 0)):
+        self.entityId = entityId
+        self.typeName = typeName
+        self.pos = pos or Pos()
+        self.in_inventory = -1
```

Comment:
we should guidelines to what the numbers mean it becomes really hard to follow really quick (like the priority stuff for tasks), for example `-1` would imply not in inventory I assume ?